code/src/java/pcgen/gui2/facade/CoreUtils.java

```diff
@@ -245,7 +245,7 @@ public String toString()
 		
 	}
 
-	private static class ObjectCoreViewNode<T> extends CoreViewNodeBase
+	private static final class ObjectCoreViewNode<T> extends CoreViewNodeBase
```

Comment:
Same as above - what message is this trying to send?  "I did this because" or "there is a design reason why you shouldn't extend this?"
The latter is valuable, the former, not so much.